### DISCUSSION TITLE
cmder

### DISCUSSION THREAD

**Discussion by gianslepetis** [Launch-X-Latam/MisionFrontEnd#5] [Q&A]
¿Es recomendable utlizar cmder o la integrada en vs code?

**Comment by magouwuxd** [comment]
usa el que gustes, por comodidad yo uso el de vs code, porque ya te pone dentro del directorio de la aplicacion

**Comment by Allegseu95** [comment]
La que viene integrada dentro de VS Code.

**Comment by doguedogue** [comment]
Para mi igual es más fácil dentro de VS Code, Pero por fuera si tener tu Terminal, powershell, bash, Terminator o la que gustes
